docs/requirements.txt

```diff
@@ -2,3 +2,18 @@ Sphinx>=2.1
 sphinx-hoverxref
 sphinx-notfound-page
 sphinx_rtd_theme
+
+# Required for ReadTheDocs
```

Comment:
I wonder if instead of duplicating the requirements we can ask readthedocs to install Scrapy using pip; this seems to be supported (see https://docs.readthedocs.io/en/stable/config-file/v2.html#packages)

Reply:
It worked, cool. TIL